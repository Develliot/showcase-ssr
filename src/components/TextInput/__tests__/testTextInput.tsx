import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import { TextInput } from '../';

it('renders correctly', () => {
    const wrapper: ShallowWrapper = shallow(
        <TextInput name='input' labelText='some label text' />
    );
    expect(wrapper).toMatchSnapshot();
});
