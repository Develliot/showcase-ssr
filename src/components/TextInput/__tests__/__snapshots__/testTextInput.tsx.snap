// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<styles__TextInputWrapper>
  <styles__InputStyled
    name="input"
    placeholder=" "
    required={false}
    type="text"
  />
  <styles__LabelStyled>
    some label text
  </styles__LabelStyled>
</styles__TextInputWrapper>
`;
